### ISSUE TITLE
rename `rpch-trail-access` DB to `marketing`

### ISSUE THREAD

**Issue by nionis** [Rpc-h/RPCh#216]

**Comment by vonlabs** [comment]
I do not feel that keeping 1 db for multiple services is a good idea, in case our .env will get leaked.

**Comment by mjadach-iv** [comment]
@nionis I do not feel that keeping 1 db for multiple services is a good idea, in case our .env will get leaked.

**Comment by nionis** [comment]
@sitilge agreed - furthermore requests to create a new DB must be documented to the epic prior to the approval of the epic (in short: don't use GCP resources without checking with the team)
